[1] - Registration Page Updated

akash-kumar-dev opened the pull request:
Addresses: #1 

## Registration Page redesigned

Reference:   [Figma Design](https://www.figma.com/design/pn2JO3bbMEzztJHuXJnS10/nise3-skill-and-certification?node-id=0-1&t=v4yFDuE6jZqnIODw-1)

![image](https://github.com/user-attachments/assets/4ddb8f54-165b-48d5-be33-94287787e396)

Shelly011s commented:
The width of the form is not similar to the one in figma design.
Similarly for the navbar fix the height. 
Input box border and background color are also not similar. 
Please be precise.

akash-kumar-dev commented:
# Global Changes
- Fixed Navbar height
- Fixed Footer x-axis scroll

# Updated Sign Up page
- Fixed Input box border from registration form
- Fixed input box background colour
- Fixed registration Form width
![image](https://github.com/user-attachments/assets/55bb6829-1cf0-4854-a30a-d29e9555dd6c)


# Updated Login Page
- Fixed Input box border from registration form
- Fixed input box background colour
- Fixed y-axis scroll
![image](https://github.com/user-attachments/assets/7e888487-b980-405c-b28b-77c14435129f)
